app.js

```diff
@@ -3,29 +3,28 @@ const express = require('express');
 require('express-async-errors');
 const logger = require('morgan');
 const cors = require('cors');
-const Sequelize = require('sequelize');
+const swaggerUi = require('swagger-ui-express');
+const yaml = require('js-yaml');
+const fs = require('fs');
 
-const app = express();
+const swaggerDocument = yaml.safeLoad(fs.readFileSync('./swagger.yaml', 'utf8'));
+const models = require('./models');
 
-// Sequelize init
-const host = process.env.RDS_HOSTNAME || 'localhost';
-const user = process.env.RDS_USERNAME || 'hrm';
-const pass = process.env.RDS_PASSWORD || '';
+const app = express();
 const apiKey = process.env.API_KEY;
-
 const version = '0.3.6';
 
 if (!apiKey) {
   throw new Error('Must specify API key');
 }
 
 console.log(`Starting HRM version ${version}`);
-console.log(`Trying with: ${[host, user].join(', ')}`);
-const sequelize = new Sequelize('hrmdb', 'hrm', pass, {
-  host,
-  dialect: 'postgres',
-});
-const ContactController = require('./controllers/contact-controller')(sequelize);
+
+app.use('/swagger', swaggerUi.serve, swaggerUi.setup(swaggerDocument));
```

Comment:
Do you think we should run swagger in production?  Feels like not a good idea, but I can be convinced.

Reply:
I agree. I've fixed it to run only when it's not production.